### PR TITLE
RIA-8104 Create Rule32 and 31

### PR DESCRIPTION
**Pull request by MiguelMartinRojas**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RIA-8104

### Change description ###

RIA-8104 Create Rule32 and 31

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmation.java [hunk 1]
```diff
@@ -53,32 +67,43 @@ public PostSubmitCallbackResponse handle(
...
                 );
                 break;
 
             case "remadeRule31":
             case "remadeRule32":
-                postSubmitResponse.setConfirmationBody(
-                    "#### What happens next\n\n"
-                    + "Both parties have been notified of the decision.<br>"
-                );
+                YesOrNo isDlrmSetAsideEnabled
```
Comment (colmlynch60) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmation.java:92]: can this be change to be consistent with how Thomas does it - 
image

**Review by colmlynch60 — COMMENTED** [review]
just a tiny change to be consistent with Thomas' implementation

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Review by ikirsanov — APPROVED** [review]

**Review by javierhmcts — APPROVED** [review]
LGTM

**Pull request merged**